code/__HELPERS/name_helpers.dm

```diff
@@ -132,21 +132,16 @@ GLOBAL_DATUM(syndicate_code_response_regex, /regex)
 		. = ""
 	else
 		. = list()
-	var/words = pick(//How many words there will be. Minimum of two. 2, 4 and 5 have a lesser chance of being selected. 3 is the most likely.
-		50; 2,
-		200; 3,
-		50; 4,
+	var/words = pick(//How many words there will be. Minimum of three. 4 is the most likely.
+		100; 3,
+		150; 4,
 		25; 5
 	)
 
 	var/safety[] = list(1,2,3)//Tells the proc which options to remove later on.
-	var/nouns[] = list("love","hate","anger","peace","pride","sympathy","bravery","loyalty","honesty","integrity","compassion","charity","success","courage","deceit","skill","beauty","brilliance","pain","misery","beliefs","dreams","justice","truth","faith","liberty","knowledge","thought","information","culture","trust","dedication","progress","education","hospitality","leisure","trouble","friendships", "relaxation")
-	var/drinks[] = list("vodka and tonic","gin fizz","bahama mama","manhattan","black Russian","whiskey soda","long island tea","margarita","Irish coffee"," manly dwarf","Irish cream","doctor's delight","Beepksy Smash","tequila sunrise","brave bull","gargle blaster","bloody mary","whiskey cola","white Russian","vodka martini","martini","Cuba libre","kahlua","vodka","wine","moonshine")
-	var/locations[] = length(SSmapping.teleportlocs) ? SSmapping.teleportlocs : drinks//if null, defaults to drinks instead.
+	var/nouns[] = list("love","hate","anger","peace","pride","sympathy","bravery","loyalty","honesty","integrity","compassion","charity","success","courage","deceit","skill","beauty","brilliance","pain","misery","belief","dream","justice","truth","faith","liberty","knowledge","thought","information","culture","trust","dedication","progress","education","hospitality","leisure","trouble","friendship","relaxation","laser","robot","box","crate","camera","wire","glass","plasma","gold","iron","titanium","diamond")
```

Comment:
could you pull these into text files in `config/names` (e.g. `config/names/codeword_nouns.txt` and `config/names/codeword_locations.txt`) and load them into a global list in `code/_globalvars/lists/names.dm`?

Reply:
I think I did it? Seems to work fine